omod/src/main/java/org/openmrs/module/coreapps/page/controller/findpatient/FindPatientPageController.java

```diff
@@ -28,6 +31,7 @@ public void get(PageModel model, @RequestParam("app") AppDescriptor app, UiSessi
         model.addAttribute("heading", app.getConfig().get("heading").getTextValue());
         model.addAttribute("label", app.getConfig().get("label").getTextValue());
         model.addAttribute("showLastViewedPatients", app.getConfig().get("showLastViewedPatients").getBooleanValue());
+		model.addAttribute("showRegistrationLink", app.getConfig().get("showRegistrationLink").getTextValue());
```

Comment:
@wluyima With the change to an app configuration parameter there is no need to have a GP. 

If the value is empty or null, then the button is not shown since the button will be broken. 

Hence my recommendation is to rename the configuration variable to registrationButtonAppId